src/App.jsx

```diff
@@ -2,14 +2,15 @@ import { Route, Routes } from "react-router-dom";
 
 import { Footer, Navbar } from "./Components";
 
-import { Home } from "./Pages";
+import { Home, MaterialSection } from "./Pages";
```

Comment:
Change the name to `Material` instead of `MaterialSection`. The name MaterialSection seems like a component instead of page.